src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import classNames from 'classnames';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    <p className={classNames('', { Person__age: person.age })}>
+      {person.age ? <p>I am {person.age}</p> : <></>}
+    </p>
+    <p className={classNames('', { Person__partner: person.partnerName })}>
+      {person.isMarried === true ? (
+        <p>
+          {person.partnerName} is my
+          {person.sex === 'm' ? ' wife' : ' husband'}
```

Comment:
Nested ternary looks complicated, let's create a separate variable and use it here

`const partner = person.sex === 'm' ? ' wife' : ' husband'`